test/WebJobs.Extensions.Tests/Extensions/Timers/Scheduling/TimerScheduleTests.cs

```diff
@@ -131,6 +132,40 @@ public void Create_InvalidSchedule()
             Assert.Equal("The schedule expression 'invalid' was not recognized as a valid cron expression or timespan string.", ex.Message);
         }
 
+        [Theory]
+        [InlineData("* * * * * *", null, true)]
+        [InlineData("*/5 * * * * *", null, true)]
+        [InlineData("  */5    * *      * * *", null, true)]
+        [InlineData("10,20,45 * * * * *", null, true)]
+        [InlineData("0 */5 * * * *", null, false)]
+        [InlineData("0 30 * * * *", null, false)]
+        [InlineData("* 30 * * * *", null, false)]
+        [InlineData("* */30 * * * *", null, false)]
+        [InlineData("0 * * * * *", null, false)]
+        [InlineData("30 * * * * *", null, false)]
+        [InlineData("* */20 * * * *", "10/30/2020 12:00:00 AM", false)]
+        [InlineData("* */20 * * * *", "10/30/2020 12:20:00 AM", false)]
+        [InlineData("* */20 * * * *", "10/30/2020 12:15:00 AM", false)]
+        [InlineData("* */20 * * * *", "10/30/2020 12:19:30 AM", false)]
+        [InlineData("* * 12 * * Mon", null, false)]
+        [InlineData("* 59 11 * * 1,2,3,4,5", null, false)]
```

Comment:
This does not consider the range definition for cron.  ex: * 1-5 * * * *,  Every second during minute :02 of 1-5 of every hour.

Reply:
That is also handled - I'll add a test case for that.